EventListener/UserContextSubscriber.php

```diff
@@ -61,21 +62,28 @@ class UserContextSubscriber implements EventSubscriberInterface
      */
     private $ttl;
 
+    /**
```

Comment:
and please explain here that not booting the symfony kernel is the reason why we use this matcher rather than just looking at symfony and see if there is a non-anonymous user.